Source/WebCore/Modules/mediastream/RTCStatsReport.h

```diff
@@ -145,54 +156,62 @@ class RTCStatsReport : public RefCounted<RTCStatsReport> {
         std::optional<uint64_t> bytesSent;
     };
 
+    enum class QualityLimitationReason {
+        None,
+        Cpu,
+        Bandwidth,
+        Other
+    };
+
     struct OutboundRtpStreamStats : SentRtpStreamStats {
         OutboundRtpStreamStats() { type = RTCStatsReport::Type::OutboundRtp; }
 
-        std::optional<uint32_t> rtxSsrc;
+        String mid;
         String mediaSourceId;
-        String senderId;
         String remoteId;
         String rid;
-        std::optional<double> lastPacketSentTimestamp;
         std::optional<uint64_t> headerBytesSent;
-        std::optional<uint32_t> packetsDiscardedOnSend;
-        std::optional<uint64_t> bytesDiscardedOnSend;
-        std::optional<uint32_t> fecPacketsSent;
         std::optional<uint64_t> retransmittedPacketsSent;
         std::optional<uint64_t> retransmittedBytesSent;
+        std::optional<uint32_t> rtxSsrc;
         std::optional<double> targetBitrate;
         std::optional<uint64_t> totalEncodedBytesTarget;
         std::optional<uint32_t> frameWidth;
         std::optional<uint32_t> frameHeight;
-        std::optional<uint32_t> frameBitDepth;
         std::optional<double> framesPerSecond;
         std::optional<uint32_t> framesSent;
         std::optional<uint32_t> hugeFramesSent;
         std::optional<uint32_t> framesEncoded;
         std::optional<uint32_t> keyFramesEncoded;
-        std::optional<uint32_t> framesDiscardedOnSend;
         std::optional<uint64_t> qpSum;
-        std::optional<uint64_t> totalSamplesSent;
-        std::optional<uint64_t> samplesEncodedWithSilk;
-        std::optional<uint64_t> samplesEncodedWithCelt;
-        std::optional<bool> voiceActivityFlag;
         std::optional<double> totalEncodeTime;
         std::optional<double> totalPacketSendDelay;
-        std::optional<double> averageRtcpInterval;
-        // std::optional<RTCQualityLimitationReason qualityLimitationReason;
-        // std::optional<record<DOMString, double> qualityLimitationDurations;
+        std::optional<QualityLimitationReason> qualityLimitationReason;
+        std::optional<Vector<KeyValuePair<String, double>>> qualityLimitationDurations;
         std::optional<uint32_t> qualityLimitationResolutionChanges;
-        // std::optional<record<USVString, unsigned long long> perDscpPacketsSent;
         std::optional<uint32_t> nackCount;
         std::optional<uint32_t> firCount;
         std::optional<uint32_t> pliCount;
-        std::optional<uint32_t> sliCount;
-        // DOMString encoderImplementation;
+        String encoderImplementation;
+        std::optional<bool> powerEfficientEncoder;
```

Comment:
This increases the size of the struct while not being used.
I would remove them until we have a plan to expose them somehow.